stdlib/socket.pyi

```diff
@@ -768,7 +768,13 @@ def getnameinfo(sockaddr: Union[Tuple[str, int], Tuple[str, int, int, int]], fla
 def getprotobyname(protocolname: str) -> int: ...
 def getservbyname(servicename: str, protocolname: str = ...) -> int: ...
 def getservbyport(port: int, protocolname: str = ...) -> str: ...
-def socketpair(family: int = ..., type: int = ..., proto: int = ...) -> Tuple[socket, socket]: ...
+
+if sys.platform == "win32":
+    def socketpair(family: int = ..., type: int = ..., proto: int = ...) -> Tuple[socket, socket]: ...
```

Comment:
Is it possible to use `family: Literal[AF_INET, AF_INET6]`? I think `Literal` should support enums.

Reply:
The default value is AF_INET which is int, not AddressFamily, so that gives an error.

Reply:
```
$ grep AF_INET: stdlib/socket.pyi
AF_INET: AddressFamily
        AF_INET: int
```

The indented one is inside an enum, meaning the value of the enum has type `int`, although the enum member doesn't.